src/main/java/org/folio/gobi/PurchaseOrderParser.java

```diff
@@ -62,4 +74,110 @@ public PurchaseOrder parse(Reader data) throws PurchaseOrderParserException {
 
     return purchaseOrder;
   }
+
+  public static class ResourceResolver implements LSResourceResolver {
+
+    @Override
+    public LSInput resolveResource(String type, String namespaceURI, String publicId, String systemId, String baseURI) {
+      InputStream resourceAsStream = this.getClass()
+        .getClassLoader()
+        .getResourceAsStream(systemId);
+      return new Input(publicId, systemId, resourceAsStream);
+    }
+  }
+
+  public static class Input implements LSInput {
+
+    private static final Logger logger = LoggerFactory.getLogger(Input.class);
```

Comment:
General question about loggers.  Can an instance of the logger be injected (instead of being obtained in the class) so that we have more control over them in the unit tests, e.g., querying the logger to see if certain errors were logged?  I suppose that the LoggerFactory could be set up to have mock loggers, but not sure how easily it is to mock out LoggerFactory objects.